cocos/renderer/CCGLProgram.h

```diff
@@ -110,6 +110,10 @@ class CC_DLL GLProgram : public Ref
         VERTEX_ATTRIB_BLEND_WEIGHT,
         /**Index 8 will be used as Blend index.*/
         VERTEX_ATTRIB_BLEND_INDEX,
+        /**Index 9 will be used as tangent.*/
+        VERTEX_ATTRIB_TANGENT,
+        /**Index 10 will be used as Binormal.*/
+        VERTEX_ATTRIB_BINORMAL,
```

Comment:
Vertex attribute must be handled carefully, because in some devices, only 8 vertex attribute is supported, we may need map `VERTEX_ATTRIB_BLEND_INDEX `, `VERTEX_ATTRIB_POSITION` dynamically.